File: exercice2/main.ts
class Furniture{
  editor: string;
  designer: string;
  creationYear: number;

  constructor(editor: string, designer: string, creationYear: number){
    this.editor = editor;
    this.designer = designer;
    this.creationYear = creationYear
  }
}

let wassily: Furniture = new Furniture("Knoll", "Marcel Breuer", 1925);
console.log(wassily.editor, wassily.designer, wassily.creationYear);
wassily.creationYear = 1926;
console.log(wassily.creationYear);

File: exercice2/main.java
class Furniture {
    String editor;
    String designer;
    int creationYear;

    Furniture(String editor, String designer, int creationYear) {
        this.editor = editor;
        this.designer = designer;
        this.creationYear = creationYear;
    }
}
class Playground {
    public static void main(String[ ] args) {
        Furniture wassily = new Furniture("Knoll", "Marcel Breuer", 1925);
        wassily.creationYear = 1926;
        System.out.println("Designed by " + wassily.designer + " in " + wassily.creationYear + ".");
        System.out.println("Edited by " + wassily.editor + ".");
    }
}
